collectd-mlab.spec

```diff
@@ -77,17 +83,41 @@ install -D -m 644 viewer/collection-mlab.conf		%{buildroot}/etc/collection-mlab.
 install -D -m 644 plugin/collectd			%{buildroot}/etc/default/collectd
 install -D -m 644 plugin/collectd-mlab.conf		%{buildroot}/etc/collectd-mlab.conf
 install -D -m 644 plugin/types.db			%{buildroot}/usr/share/collectd-mlab/types.db
-install -D -m 644 plugin/mlab.py			%{buildroot}/var/lib/collectd/python/mlab.py
+install -D -m 644 plugin/mlab.py			%{buildroot}/var/lib/collectd/python/mlab_vs.py
+
+# Init script.
+install -D -m 755 plugin/collectd-setup		%{buildroot}/etc/init.d/collectd-setup
+
+# Python site-packages modules.
+install -d %{buildroot}/%{site_packages}/mlab/disco
+install -D -m 644 site-packages/mlab/disco/__init__.py \
+                  site-packages/mlab/disco/arp.py \
+				  site-packages/mlab/disco/collectd.py \
+                  site-packages/mlab/disco/discovery.py \
+                  site-packages/mlab/disco/models.py \
+                  site-packages/mlab/disco/network.py \
+                  site-packages/mlab/disco/route.py \
+                  site-packages/mlab/disco/simple_session.py \
+				  %{buildroot}/%{site_packages}/mlab/disco
+
+# Disco commands.
+install -D -m 755 disco/disco_config.py	%{buildroot}/usr/bin/disco_config.py
+
+# Disco config.
+install -D -m 755 disco/models.yaml \
+    %{buildroot}/usr/share/collectd-mlab/models.yaml
```

Comment:
Can /models.yaml be removed from the end of this destination, since it's the same name as the origin file?